CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+Contributing to MockBukkit
+==========================
+
+intro here
+
+
+## Requirements
```

Comment:
Probably good to clarify here that these requirements are software-related and not skill, or legal-based.

```suggestion
## :toolbox: Software requirements
```